vendor/parser-extended/ast_mapper.ml

```diff
@@ -15,10 +15,8 @@
 
 (* A generic Parsetree mapping class *)
 
-(*
```

Comment:
This warning being commented out was the pain source of a bug for a couple hours. I don't wee why it's commented out when there's only like 2 places below that needed to be changed to make the warning pass.